storeshare_api/app/models/listing.rb

```diff
@@ -1,6 +1,18 @@
 # frozen_string_literal: true
 
 class Listing < ApplicationRecord
+  include Filterable
+
   belongs_to :subletter
   has_one_attached :image, dependent: :destroy
+
+  scope :filter_by_city, ->(city) { where city: city }
+  scope :filter_by_state, ->(state) { where state: }
```

Comment:
should this line be `(state) { where state: state }` ?

Reply:
yeah, I wonder why the code didn't crash lol